docs/5.0/pages/quickstart.md

```diff
@@ -19,17 +19,20 @@ Teleport on Linux machine(s).
 There are several ways to install Teleport.
 Take a look at the [Teleport Installation](installation.md) page to pick the most convenient for you.
 
-=== "yum repo / AWS Linux 2"
+=== "yum repo"
 
     ```bash
     sudo yum-config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
     sudo yum install teleport
-
-    # Optional:  Using DNF on newer distributions
-    # sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-    # sudo dnf install teleport
```

Comment:
They've actually removed yum completely on newer versions of Fedora, so if if isn't too much of a pain I'd suggest leaving this in.

Reply:
Good call, I've added it back.